src/app/icd/client/DefaultICDClientStorage.cpp

```diff
@@ -39,6 +40,10 @@ static_assert(kMaxFabricListTlvLength <= std::numeric_limits<uint16_t>::max(), "
 
 namespace chip {
 namespace app {
+
+inline constexpr uint32_t kCheckInCounterMax       = UINT32_MAX;
+inline constexpr uint32_t kCheckInRolloverConstant = (1U << 31);
```

Comment:
```suggestion
inline constexpr uint32_t kKeyRefreshLimit = (1U << 31);
```
or some version that informs of the check.